src/state.js

```diff
@@ -0,0 +1,73 @@
+import Project from './project'
+import Task from './task'
+
+let db = firebase.firestore()
+const docRef = db.collection('projects').doc('Naqhid')
+
+export default function saveState() {
+  // localStorage.setItem('projects', JSON.stringify(window.projects));
+  docRef
+    .withConverter(projectConverter)
+    .set(window.projects)
+  .then(() => {
+    console.log('saved.')
```

Comment:
No `console.log` committed to the repo. It's ugly, it kills performance. and it can make confidential data be visible in the browser tools.
 
 The previous reviewer @rindrajosia  meant you should remove this `console.log`.